content/en/agent/guide/integration-management.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Integration Management
+kind: guide
+---
+
+## Overview
+
+The Agent comes with a set of bundled official Datadog integrations to allow users to quickly start monitoring their applications. These integrations are also available as single Python packages that can be individually upgraded.
```

Comment:
```suggestion
The Agent comes with a set of bundled official Datadog integrations to allow users to start monitoring their applications quickly. These integrations are available as single Python packages, and you can upgrade them separately.
```

Reply:
the first sentence is a split infinitive and the second sentence is passive voice that I think could a bit more clear when made active